_inc/lib/debugger/class-jetpack-cxn-tests.php

```diff
@@ -362,6 +363,47 @@ protected function test__server_port_value() {
 		}
 	}
 
+	/**
+	 * If Sync is enabled, this test will be skipped. If Sync is disabled, the test will fail.
+	 * Eventually, we'll make this test more robust with additional states. Here is the plan for possible Sync states,
+	 * including states that are planned but not yet implemented.
+	 *
+	 * Enabled: Skips test
+	 * Disabled: Results in a failing test
+	 * Healthy: @todo
+	 * In Progress: @todo
+	 * Delayed: @todo
+	 * Error: @todo
+	 */
+	protected function test__sync_health() {
+		$name = __FUNCTION__;
+		if ( ! $this->helper_is_jetpack_connected() ) {
+			// If the site is not connected, there is no point in testing Sync health.
+			return self::skipped_test( $name, false, false );
+		}
+		if ( Sync_Settings::is_sync_enabled() ) {
+			return self::skipped_test( $name, false, false );
+		}
+		return self::failing_test(
+			$name,
+			__( 'Jetpack Sync has been disabled on your site.', 'jetpack' ),
+			'enable_sync',
+			admin_url( '#' ),
```

Comment:
Are we sure we want to land on the Admin page in this case. It might be worth checking if we have a end user visible guide on Sync, how to enable, or etc?

Reply:
that's an outstanding question. i have no idea where to send people for this.

Reply:
@roccotripaldi You mentioned there being a filter to turn off sync. Is this documented anywhere in our support pages? If so, we should link it there. If not, maybe JPOP Happy can create a document for it?